src/mmc.c

```diff
@@ -193,21 +201,20 @@ int mmc_init(unsigned int id)
 
 	/* 0x100 -> VHS in 2.7-3.6V
 	 * 0x0aa -> pattern to read back */
-	ret = mmc_cmd(id, CMD_SEND_IF_COND, 0x1aa, 0x0, MSC_RESPONSE_R7, resp);
-	if (ret) {
-		SERIAL_ERR(ret);
-		return ret;
-	}
-
-	if (resp[0] != 0x1aa)
-		return ERR_MMC_INIT;
+	mmc_cmd(id, CMD_SEND_IF_COND, 0x1aa, 0x0, MSC_RESPONSE_R7, resp);
 
 	for (retries = 1000; retries; retries--) {
-		mmc_cmd(id, CMD_APP_CMD, 0, 0x0, MSC_RESPONSE_R1, resp);
+		ret = mmc_cmd(id, CMD_APP_CMD, 0, 0x0, MSC_RESPONSE_R1, resp);
+
```

Comment:
Remove the empty line here (and every time you check (ret) below).

Reply:
done